pkg/kubectl/cmd/util/clientcache.go

```diff
@@ -17,30 +17,36 @@ limitations under the License.
 package util
 
 import (
+	fed_client_1_3 "k8s.io/kubernetes/federation/client/clientset_generated/release_1_3"
```

Comment:
@caesarxuchao yeah, I think you are right. Sorry, I wasn't in the room when that conversion happened.